orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/DependentPipelineStarter.groovy

```diff
@@ -105,9 +122,24 @@ class DependentPipelineStarter implements ApplicationContextAware {
     }
 
     log.info('executing dependent pipeline {}', pipeline.id)
+    return pipeline
+  }
 
-    pipeline
+  // There are currently two sources-of-truth for the user:
+  // 1. The MDC context, which are the values that get propagated to downstream services like Front50.
+  // 2. The Execution.AuthenticationDetails object.
```

Comment:
FWIW the MDC context is set based on the Execution.AuthenticationDetails each time a task run (via `TaskTasklet`).